dll/win32/shell32/lang/de-DE.rc

```diff
@@ -1003,4 +1003,8 @@ BEGIN
     IDS_COPYTOTITLE "Wählen Sie den Ort, in den '%s' kopiert werden soll. Klicken Sie anschließend auf Kopieren."
     IDS_COPYITEMS "Elemente kopieren"
     IDS_COPYBUTTON "Kopieren"
+    IDS_MOVETOMENU "Move to &folder..."
+    IDS_MOVETOTITLE "Select the place where you want to move '%s' to. Then click the Move button."
```

Comment:
IDS_MOVETOTITLE "Wählen Sie den Ort, in den '%s' verschoben werden soll. Klicken Sie anschließend auf Verschieben."

Reply:
OK.